pkg/workflowengine/impl/propeller_executor.go

```diff
@@ -103,6 +106,42 @@ func addExecutionOverrides(taskPluginOverrides []*admin.PluginOverride,
 	if workflowExecutionConfig != nil {
 		executionConfig.MaxParallelism = uint32(workflowExecutionConfig.MaxParallelism)
 	}
+	if taskResources != nil {
+		var requests = v1alpha1.TaskResourceSpec{}
+		if taskResources.Defaults != nil {
+			if len(taskResources.Defaults.Cpu) > 0 {
+				requests.CPU = resource.MustParse(taskResources.Defaults.Cpu)
```

Comment:
I guess it's too late to ask for to change the config to use resource.Quantity instead of a string?

Reply:
ah I can deprecate the string values perhaps